packages/cli/src/commands/certs/ls.ts

```diff
@@ -43,10 +43,7 @@ async function ls(
   }
 
   // Get the list of certificates
-  const { certs, pagination } = await getCerts(
-    client,
-    ...paginationOptions
-  ).catch(err => err);
+  const { certs, pagination } = await getCerts(client, ...paginationOptions);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** Can we add a test for the failure case here?

Reply:
good point; I'll add a test for it. 🙌

Reply:
@EndangeredMassa 

error message if the "fix" would not be in place:

<img width="712" alt="image" src="https://github.com/vercel/vercel/assets/2198240/f61a4783-d903-43f8-9587-f4414e3d07fa">

https://github.com/vercel/vercel/pull/10551/commits/02327c95d17311b03c94722f6339f99545c663c7